### DISCUSSION TITLE
Telephone Number Field

### DISCUSSION THREAD

**Discussion by bengreenbank** [craftcms/cms#9812] [Ideas]
Rather than having to use the number field, it would be good to have a telephone number field, which could allow you to select whether a specific dialing code (+44 etc) must be entered, or whether an extension is allowed.

Currently it can cause issues to use the number field for this, as some countries numbers are longer than others, so it's not possible to set a max length and the number could have an extension.

**Comment by andris-sevcenko** [comment]
I personally haven't used it, but there's a plugin for that - https://plugins.craftcms.com/phone-number

**Reply by adrienne** [reply]
There's actually a couple. I personally like Leevi Graham's [NSM Fields](https://plugins.craftcms.com/nsm-fields), but [Sprout Fields](https://plugins.craftcms.com/sprout-fields) is also good.
